src/util/getOptimisticIndex.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-bitwise */
+
+import { Message } from '@graphql/graphql';
+import { StateMessages } from 'types/messages.types';
+
+export function getOptimisticIndex(messages: StateMessages[], recentMessage: Message) {
+  return messages.findIndex(
```

Comment:
```suggestion
  return messages.findIndex(m =>
```

Use braces instead with a return statement, it'll be more readable